### PR TITLE
Fix 02

### PR DESCRIPTION
**Pull request by ronaldpabilonia**

### CODE REVIEW

**Comment by ronaldpabilonia** [comment]
@adin234 ignore the other pull request, this one is much better thanks :)

**Comment by ronaldpabilonia** [comment]
@adin check again please :) i've also updated https://github.com/anyTV/heartbeat-comments-extension/pull/127 to conform changes here

File: app.js [hunk 1]
```diff
@@ -2,53 +2,54 @@ var app = require('http').createServer(handler),
...
 
-function start_iteration (data) {
-  recheck = function () {
-    find_streamers(data);
-    setTimeout(recheck, 900000);
-    // 900000 = 15 minutes (check streamers if they are online every 15 minutes)
-  };
+io.on('connection', function (socket) {
+  socket.on('user_favorite_streamers', function (data) {
+    streamers.push(data);
```
Comment (adin234) [inline app.js]: will you please give us an example of `data`

Comment (ronaldpabilonia) [inline app.js]: http://gyazo.com/bb80dce3ea80a555629afd89a0c43871

i made the data into string to save memory

**Comment by ronaldpabilonia** [comment]
@nhvu1988  up

**Comment by ronaldpabilonia** [comment]
@adin234 who's supposed to merge this?

**Comment by ravenjohn** [comment]
@ronaldpabilonia why are you using 2 space indention

**Comment by ronaldpabilonia** [comment]
@ravenjohn ok let me fix that

**Comment by ronaldpabilonia** [comment]
@ravenjohn ready

File: app.js [hunk 2]
```diff
@@ -48,7 +24,33 @@ function handler (req, res) {
...
+                .then(function (err, result) {
+                    if (io.engine.clients) {
+                        if (err) {
+                            io.emit('online_streamers', err);
+                        }
+
+                        io.emit('online_streamers', result);
+                    }
+                });
+        }).
```
Comment (ravenjohn) [inline app.js]: ...

Comment (ronaldpabilonia) [inline app.js]: i'm not supposed to emit the error?

Comment (ronaldpabilonia) [inline app.js]: anybody there? D:

Comment (ravenjohn) [inline app.js]: https://github.com/anyTV/JS-conventions/blob/master/lodash.md#pattern

**Comment by ronaldpabilonia** [comment]
up

**Pull request merged**